src/tests/uat/process_search_calls.py

```diff
@@ -407,33 +544,31 @@ def main():
     parser.add_argument("--window", help="Define search window", default='3d')
 
     args = parser.parse_args()
-    print_detail = args.verbose
-    window = '-' + args.window
-
-    do_process = True
-    do_enriched_events = True
+    global ORG_KEY
+    global HOSTNAME
 
-    if print_detail:
-        print(f"profile being used is {args.__dict__}")
+    print_detail = False
```

Comment:
It seems the only way to use the print_detail is to change it in the code, which is not optimal. We should either remove it, or just add another parser.argument for it just like the --window one.

Reply:
Jira ticket created to cover this issue in all UAT tests where applicable: CBAPI-2613